lib/Basics/StringBuffer.cpp

```diff
@@ -826,11 +817,12 @@ int arangodb::basics::StringBuffer::inflate(std::stringstream& out,
       TRI_ASSERT(res != Z_STREAM_ERROR);
 
       switch (res) {
-      case Z_NEED_DICT:
-      case Z_DATA_ERROR:
-      case Z_MEM_ERROR: {
-        return TRI_ERROR_INTERNAL;
-      }
+        case Z_NEED_DICT:
+        case Z_DATA_ERROR:
+        case Z_MEM_ERROR: {
+          return TRI_ERROR_INTERNAL;
+        }
+        default:;
```

Comment:
should work, but looks a bit ugly. do we need this to suppress compiler warnings?

Reply:
Not compiler, but a clang-tidy warning `hicpp-multiway-paths-covered`.

As to the uglyness, would you prefer
```suggestion
        default: {
        }
```
?

Reply:
It is not important for me. It just looked a bit strange to me. I am fine with any outcome.